src/stedi/cdk/lambda/build.clj

```diff
@@ -84,7 +84,7 @@
 (defn- build-lib-layer
   [deps-map build-dir]
   (let [deps-map* (-> deps-map
-                      (select-keys [:deps])
+                      (select-keys [:deps :mvn/repos])
```

Comment:
Theres actually a much better resolution mechanism here https://github.com/clojure/tools.deps.alpha/blob/master/src/main/clojure/clojure/tools/deps/alpha/reader.clj#L100-L106 that I've been meaning to use instead.